07.bitstream.semantics.md

```diff
@@ -1314,6 +1315,41 @@ coded video sequence.
 number of bits used to represent time_offset_value is equal to time_offset_length. When
 time_offset_value is not present, its value is inferred to be equal to 0.
 
+#### Metadata crop semantics
+
+When present the metadata_crop OBU applies starting at the next frame in the sequence 
+with matching temporal_id and spatial_id, and shall apply to all matching frames until the next Key Frame
+or metadata_crop OBU. The output picture should be cropped to the region as specified in the 
+cropping metadata OBU. When applied, the crop shall be after all normal decode operations as a 
+post-processing step (after film grain synthesis in the output process).  This metadata has no 
+effect on the decoding process.
+
+**crop_x_offset** specifies the minimum pixel column containing picture data which should be rendered.
+
+**crop_y_offset** specifies the minimum pixel row containing picture data which should be rendered.
+
+**crop_width_minus_1** specifies the number of pixel columns minus 1 which which should be rendered after applying crop_x_offset.
+
+**crop_height_minus_1** specifies the number of pixel rows minus 1 which which should be rendered after applying crop_y_offset.
+
+If RenderWidth or RenderHeight are present in the frame header, then render width and height 
+shall be applied after cropping. 
+
+The total cropped area shall not exceed the frame size:
+
+ * crop_width_minus_1 + crop_x_offset shall be less than or equal to frame_width_minus_1.
+ * crop_height_minus_1 + crop_y_offset shall be less than or equal to frame_height_minus_1.
+
+The crop area shall align with the chroma subsampling grid:
+
+ * crop_x_offset and crop_width_minus_1 + 1 must both be a multiple of subsampling_x + 1.
+ * crop_y_offset and crop_height_minus_1 + 1 must both be a multiple of subsampling_y + 1.
+
+When muxed into a container that supports signaling cropping information, this metadata should 
+be removed from the bitstream and included in the container’s signaling mechanism.  
+If both the container and bitstream signal cropping information, then the container’s cropping 
+information takes precedence.
```

Comment:
Instead of:

> When muxed into a container that supports signaling cropping information, this metadata should 
be removed from the bitstream and included in the container’s signaling mechanism.  
If both the container and bitstream signal cropping information, then the container’s cropping 
information takes precedence.

We could write:
> In cases where cropping information is present in both the bitstream and the delivery or container format, the latter should be prioritized. Container or delivery formats that package the AV1 bitstream are recommended to address this redundancy, potentially by excluding bitstream cropping information when it is also available in the container or delivery format.

Then we will have to update the [AV1 ISOBMFF](https://aomediacodec.github.io/av1-isobmff/) spec and say there that we use the `clap` box and remove this metadata OBU. Thoughts @cconcolato ?

Reply:
Updated to new wording.